allensdk/brain_observatory/behavior/data_objects/metadata/subject_metadata/mouse_id.py

```diff
@@ -9,17 +9,19 @@
 class MouseId(DataObject, LimsReadableInterface, JsonReadableInterface,
               NwbReadableInterface):
     """the LabTracks ID"""
-    def __init__(self, mouse_id: int):
+    def __init__(self, mouse_id: str):
+        if not isinstance(mouse_id, str):
```

Comment:
This goes against the type. It expects string, so if it is not string, then the caller is passing the wrong type and it should be corrected there. Or the type needs to be updated.

Reply:
Mostly this was to enforce the type and avoid unittest failures from elsewhere, specifically, from_json calls on ecephys tests. I'll remove the line and see if I can find where the typing is going wrong there specifically.

Reply:
Actually, hold on, would you be cool if I moved this type check to the `from_json` method?

Reply:
yep, that works